astronomer/providers/utils/sensor_util.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from airflow.exceptions import AirflowException, AirflowSkipException
+from airflow.sensors.base import BaseSensorOperator
+
+if TYPE_CHECKING:  # pragma: no cover
+    from astronomer.providers.utils.typing_compat import Context
+
+
+def poke(cls: BaseSensorOperator, context: Context) -> bool:  # pragma: no cover
```

Comment:
May I know why we mark them as no cover?

Reply:
tests are missing for a few lines. I have reduced the scope.